Sources/NostrSDK/RelayResponse.swift

```diff
@@ -29,49 +29,55 @@ enum RelayResponse: Decodable {
 
     enum MessageType: String, Codable {
         case event = "EVENT"
-        case notice = "NOTICE"
-        case eose = "EOSE"
         case ok = "OK"
-        case count = "COUNT"
+        case eose = "EOSE"
+        case closed = "CLOSED"
+        case notice = "NOTICE"
         case auth = "AUTH"
+        case count = "COUNT"
     }
     
-    struct OKMessage {
-        let type: OKMessageType
-        let message: String?
+    struct RelayResponseMessage {
```

Comment:
Because this struct is defined inside the scope of `RelayResponse` you can simply call this one `Message`. Then external to the `RelayResponse` type, this struct will be exposed as `RelayResponse.Message`. The same would be true of the prefix if you defined it inside this `Message` type. You could call it simply `Prefix`, and it would then be `RelayResponse.Message.Prefix`. This is more in alignment with how Apple defines its own types.

Reply:
Nice feedback. Fixed!